cmd/krel/cmd/obs_stage.go

```diff
@@ -0,0 +1,194 @@
+/*
+Copyright 2020 The Kubernetes Authors.
```

Comment:
```suggestion
Copyright 2023 The Kubernetes Authors.
```

Reply:
I’ll fix it in the PR for OBS release workflow.